File: RP3/Desafio3.java
import java.util.Scanner;

public class Desafio3 {
    public static void main(String[] args) {

        try (Scanner sc = new Scanner(System.in)) {

            double soma = 0;

            for (int i = 1; i <= 5; i++) {
                System.out.print("Digite a nota " + i + ": ");
                double nota = sc.nextDouble();
                soma += nota;
            }

            double media = soma / 5;

            System.out.println("Média final: " + media);
        }
    }
}

File: RP3/Desafio3.cpp
#include <iostream>
using namespace std;

int main() {
    double soma = 0;

    for (int i = 1; i <= 5; i++) {
        double nota;
        cout << "Digite a nota " << i << ": ";
        cin >> nota;
        soma += nota;
    }

    double media = soma / 5;

    cout << "Média final: " << media << endl;

    return 0;
}